src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Acquisition/MissingSetupComponentRegistrationService.cs

```diff
@@ -90,9 +130,17 @@ public void RegisterMissingWorkloads(Guid projectGuid, ConfiguredProject project
             UnregisterProjectConfiguration(projectGuid, project);
         }
 
-        public void RegisterMissingSdkRuntimeComponentId(Guid projectGuid, ConfiguredProject project, string runtimeComponentId)
+        public void RegisterPossibleMissingSdkRuntimeVersion(Guid projectGuid, ConfiguredProject project, string runtimeVersion)
         {
-            _projectGuidToRuntimeDescriptorMap.GetOrAdd(projectGuid, runtimeComponentId);
+            // Workaround to fix #AB1460328.
```

Comment:
Please provide full link to the issue since that issue number format is specific to which Azure DevOps org it is part of. Meaning, someone in the future (or anyone else looking at the code) might not know how to use this value.

Reply:
done